### ISSUE TITLE
Training on very different dataset

### ISSUE THREAD

**Issue by CodeJjang** [jwyang/faster-rcnn.pytorch#86]
I have a dataset of images with resolution of 4k, oriented bounding boxes, and totally different labels, which I want to train this model along with resnet101 (NOT PRETRAINED) on.  
 I've read issues around regarding how to train this model on custom dataset, but I still have several questions:

1. The dataset is tagged in a custom csv format. Would you recommend to write an adapter for COCO format, PASCAL_VOC format or ImageNet format?  
I've seen that the COCO format here requires some `minival` json files which I do not how how to reproduce, but the COCO format seems the most easy one because it works with jsons.  
What would you recommend to do?

2. Will I have problem with the size of the images?

3. Regarding coarse and fine grained classes I have a total of 4 coarse grained classes, two of them (large and small vehicles) have more fine-grained classes.  
How would you recommend to implement this?

4. Regarding the oriented bounding boxes, I just consider to wrap them with a normal bounding boxes.

5. Are there any normalizations I will have to implement on the dataset? I saw that they are required when you use pretrained models, but I'm planning to retrain evertything from scratch.

6. Are there any more surprises you expect during the process which I should take care of?

Thanks
